src/types/nitro.ts

```diff
@@ -278,7 +281,14 @@ export interface NitroOptions extends PresetOptions {
   wasm?: UnwasmPluginOptions;
   experimental?: {
     legacyExternals?: boolean;
-    openAPI?: boolean;
+    openAPI?:
+      | boolean
+      | {
+          title?: string;
+          description?: string;
+          version?: string;
+          scalar?: ReferenceConfiguration;
```

Comment:
We need to namespace this. Something like `{ meta: {}, ui: { scalar: {} }` ideas welcome

Reply:
I like meta and ui! Let’s go with that